source/vulkan/runtime_vk.cpp

```diff
@@ -1183,7 +1226,10 @@ bool reshade::vulkan::runtime_vk::init_technique(technique &info, VkShaderModule
 				num_depth_attachments = 1;
 				const uint32_t attach_idx = num_color_attachments;
 
-				attachment_views[attach_idx] = _default_depthstencil_view;
+				if (_depthstencil_image_view == VK_NULL_HANDLE)
+					attachment_views[attach_idx] = _default_depthstencil_view;
+				else
+					attachment_views[attach_idx] = _depthstencil_image_view;
```

Comment:
This is incorrect. Should always be `_default_depthstencil_view`, like it was before. ReShade effects are only allowed to render in the special depth-stencil created by ReShade (which only the stencil is used of, see also implementation in other APIs).

Reply:
So, i need to make _default_depthstencil_view point to the depthstencil image replacement in the create_depthstencil_replacement method. What happens if the image format does not match with it ? ?

Reply:
This attachment is for ReShade effects that use stenciling, like SMAA. It has nothing to do with depth buffer input. So nothing to modify here.

Reply:
Oh. I see... I confused this code with another. Reverted...